cfg/2.0.0/definitions.yaml

```diff
@@ -7221,8 +7339,8 @@ groups:
       audit: "stat /etc/cron.deny"   
       tests:
         test_items:
-        - flag: "stat: cannot stat '/etc/cron.deny': No such file or directory"
-          set: true                        
+        - flag: "File: /etc/cron.deny"
```

Comment:
I don't think this change is correct: the benchmark is recommending that `/etc/cron.deny` shouldn't exist. Isn't what you had before going to work?

Reply:
apparently it didn't work perfectly before, so I thought about going with the Process of elimination, this is why I switched the set flag into false as well.

Reply:
If I run stat on a file that doesn't exist, I get: 

```
stat /etc/cron.deny
stat: cannot stat '/etc/cron.deny': No such file or directory
```

And if I run on a file that does exist, I get this: 
```
root@cfs:~# stat /etc/at.deny
  File: '/etc/at.deny'
  Size: 144       	Blocks: 8          IO Block: 4096   regular file
Device: 801h/2049d	Inode: 1376        Links: 1
Access: (0640/-rw-r-----)  Uid: (    0/    root)   Gid: (    1/  daemon)
Access: 2017-08-31 09:01:33.915066238 +0000
Modify: 2016-01-14 22:35:27.000000000 +0000
Change: 2017-08-31 09:08:20.055810029 +0000
 Birth: -
```

As the test currently stands I don't think it will work because of the single quote marks (which are present in the output, but are missing in the flag definition): 

```
root@cfs:~# stat /etc/at.deny | grep "File: /etc/at.deny"
root@cfs:~#
root@cfs:~# stat /etc/at.deny | grep "File: '/etc/at.deny'"
  File: '/etc/at.deny'
```

So this could be fixed by adding the single quotes, but I think there is a safer way: looking for "No such file or directory". Where the file shouldn't exist, "No such file or directory" should be `set: true` (and `false` if the file should exist).  Wdyt?

Reply:
Will check it again then :).
I think that with check if not exist this way:
> stat: cannot stat '/etc/cron.deny': No such file or directory

the output is considered as error and redirected to sdterr and that way I did it this way, but I will check it again

Reply:
OK fixed in https://github.com/aquasecurity/bench-common/pull/58 and changed the yaml